apps/govquests-api/rails_app/db/migrate/20241001131854_create_quests.rb

```diff
@@ -0,0 +1,13 @@
+class CreateQuests < ActiveRecord::Migration[8.0]
+  def change
+    create_table :quests do |t|
+      t.string :img_url
+      t.string :title
+      t.string :reward_type
+      t.text :intro
+      t.text :steps
+
+      t.timestamps
+    end
```

Comment:
this will probably change, right?

idk how you've thought on the concept of the Quest, but it potentially has startDatetime and EndDatetime

Reply:
Yes, I've just put what we have on the wireframe, only for starting using rails and to be able to have something on the frontend too. But today I'll insert some more data, then I'll include the startDateTime and endDateTime